README.md

```diff
@@ -140,6 +140,64 @@ by user is not implemented [#324].
 
 [#324]: https://github.com/Taxel/PlexTraktSync/issues/324
 
+## Docker Compose
```

Comment:
duplicate docker compose section. and this section isn't even about docker compose.